lib/rules/hidden-content.js

```diff
@@ -0,0 +1 @@
+return axe.commons.dom.isVisible(node);
```

Comment:
This only checks for `aria-hidden` if the `true` parameter is set as the second parameter. Am I understanding the docs correctly?